interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+			<input id="EYES5" name="EYES5" type="checkbox" value="on" >
+			<label for="EYES5"> <?php echo xlt('Eye Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES6" name="EYES6" type="checkbox" value="on" >
+			<label for="EYES6"> <?php echo xlt('Photophobia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES7" name="EYES7" type="checkbox" value="on" >
+			<label for="EYES7"> <?php echo xlt('Red Eyes');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="EYES_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+
+	
+	
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('ENT:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="ENT1" name="ENT1" type="checkbox" value="on" >
+			<label for="ENT1"> <?php echo xlt('Hearing Loss');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT2" name="ENT2" type="checkbox" value="on" >
+			<label for="ENT2"> <?php echo xlt('Ear Ache');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT3" name="ENT3" type="checkbox" value="on" >
+			<label for="ENT3"> <?php echo xlt('Otorrhea');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT4" name="ENT4" type="checkbox" value="on" >
+			<label for="ENT4"> <?php echo xlt('Tinnitis');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT5" name="ENT5" type="checkbox" value="on" >
+			<label for="ENT5"> <?php echo xlt('Epistaxis');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT6" name="ENT6" type="checkbox" value="on" >
+			<label for="ENT6"> <?php echo xlt('Sneezing');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT7" name="ENT7" type="checkbox" value="on" >
+			<label for="ENT7"> <?php echo xlt('Nasal Drip');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT8" name="ENT8" type="checkbox" value="on" >
+			<label for="ENT8"> <?php echo xlt('Sore Throat');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT9" name="ENT9" type="checkbox" value="on" >
+			<label for="ENT9"> <?php echo xlt('Mouth Dryness');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT10" name="ENT10" type="checkbox" value="on" >
+			<label for="ENT10"> <?php echo xlt('Problem Swallowing');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT11" name="ENT11" type="checkbox" value="on" >
+			<label for="ENT11"> <?php echo xlt('Dental Problems');?></label>
+		</span>
+		</TD>	
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="ENT_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+	
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('RESP:');?></B>
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation